build.gradle

```diff
@@ -67,11 +71,12 @@ subprojects {
 }
 
 javaVersions {
-    libraryTarget = 17
+    libraryTarget = 21
+    runtime = 21
 }
 
 jdks {
-    daemonTarget = 17
+    daemonTarget = 21
```

Comment:
Do we need daemonTarget 21?

Reply:
Not really, but since everything else would use java 21, I thought it is OK to move the daemon as well.